packages/image_picker/image_picker/example/lib/main.dart

```diff
@@ -40,7 +40,7 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   PickedFile _imageFile;
   dynamic _pickImageError;
-  bool isVideo = false;
+  RetrieveType mediaSource = RetrieveType.video;
```

Comment:
I'd call this `mediaType` to not confuse it with the current `source` (gallery vs camera)